src/Confluent.Kafka/Consumer.cs

```diff
@@ -194,18 +194,37 @@ public void Unsubscribe()
             => consumer.Unsubscribe();
 
         /// <summary>
-        ///     Update the assignment set to \p partitions.
+        ///     Update the assignment set to <param name="partitions" />.
         ///
-        ///     The assignment set is the set of partitions actually being consumed
-        ///     by the KafkaConsumer.
-        /// </summary>
-        /// <remarks>
-        ///     blocks
-        /// </remarks>
+        ///     The assignment set is the complete set of partitions to consume
+        ///     from.
+        /// </summary>
+        /// <param name="partitions">
+        ///     The set of partitions to consume from. If an offset value of
+        ///     -1001 (Offset.Invalid) is specified for a partition, consumption
```

Comment:
I'm wondering if Offset.Unset would be a better name than Offset.Invalid - Invalid is not a great name here. I guess the other main place this is used is in delivery reports. There, I think Invalid is better but Unset is ok. thoughts on changing this? one negative is it would be a departure from librdkafka (though i don't see that as a deal breaker). alternatively we could provide both .Invalid and .Unset but i think that's messy and potentially confusing as people would naturally assume they're different. Or the name could be .InvalidOrUnset. Or we leave as is.